engine/src/flutter/impeller/typographer/font_glyph_pair.h

```diff
@@ -54,16 +55,35 @@ struct ScaledFont {
   };
 };
 
+enum SubpixelPosition : uint8_t {
+  kSubpixel00 = 0x0,
```

Comment:
Lets document this a bit more: which ones are the X positions and which ones are Y positions?

Reply:
done